core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -260,21 +256,31 @@ private DeciderOutcome decide(final WorkflowDef workflowDef, final Workflow work
         return Collections.singletonList(rerunFromTask);
     }
 
-    private void updateOutput(final WorkflowDef def, final Workflow workflow) {
-
+    void updateOutput(final WorkflowDef def, final Workflow workflow, @Nullable Task task) {
```

Comment:
@VisibleForTesting

Reply:
package-private since it is also used from `WorkflowExecutor`.

Reply:
Added javadoc.